testing/solomachine.go

```diff
@@ -516,14 +516,12 @@ func (solo *Solomachine) GenerateClientStateProof(clientState exported.ClientSta
 	data, err := clienttypes.MarshalClientState(solo.cdc, clientState)
 	require.NoError(solo.t, err)
 
-	merklePath := solo.GetClientStatePath(clientIDSolomachine)
-	key, err := merklePath.GetKey(1) // index 0 is the key for the IBC store in the multistore, index 1 is the key in the IBC store
-	require.NoError(solo.t, err)
+	path := []byte(host.FullClientStatePath(clientIDSolomachine))
```

Comment:
don't these path functions have corresponding "key" functions which just convert them to bytes as we are doing here?